content/docs/guides/crosswalk/kubernetes/introduction/01-architecture.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Production Architecture for Teams
+
+aliases: ["/docs/guides/crosswalk/kubernetes/architecture/"]
+---
+
+Infrastructure misconfiguration accounts for the most significant subset of serious
+production outages in major systems. Some [reasonable estimates][post-mortems] put the fraction in
+the range of 40-50%.
```

Comment:
This opening feels a little out of place to me and has an inconstant tone compared to most of our other documentation. (It feels more like what you'd find in a blog post or informal essay.) Thoughts?

(also curious what @spara thinks)

Reply:
Related to tone, I do find about 1/3 of this content a bit "preachy," FWIW.

Reply:
I've toned down the preachiness and merged this into the index of Crosswalk for Kubernetes to further simplify the org structure.